### PR TITLE
Add starter issue template for github

### PR DESCRIPTION
**Pull request by jwillemsen**

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+### Version
+
+ACE: [your version]
+TAO: [your version]
+
+### Host machine and operating system
+
+If on Windows based OS's, which version of WINSOCK do you use?:
```
Comment (mitza-oci) [inline issue_template.md]: This part is no longer necessary

File: issue_template.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+### Target machine and operating system (if different from host)
+
+### Compiler name and version (including patch level)
+
+### The $ACE_ROOT/ace/config.h file 
+
+If you use a link to a platform-specific file, simply state which one]
```
Comment (mitza-oci) [inline issue_template.md]: unbalanced ]

File: issue_template.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    
+### The $ACE_ROOT/include/makeinclude/platform_macros.GNU file
+
+if you use a link to a platform-specific file, simply state which one (unless this isn't used in this case, e.g., with Microsoft Visual C++)]:
```
Comment (mitza-oci) [inline issue_template.md]: unbalanced ]

File: issue_template.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+### Synopsis
+
+Brief description of the problem
+
+### Description
+
+Detailed description of problem.  Don't just say "<blah> doesn't work, here's a fix," explain what your program does to get to the <blah> state.
+
+### Repeast by
```
Comment (mitza-oci) [inline issue_template.md]: "Repeat"

**Comment by jwillemsen** [comment]
Addressed comments

**Pull request merged**